dist/components/product/_product.dust

```diff
@@ -0,0 +1,38 @@
+{#product}
+<div class="c-product {?isHighlight}c--highlight{/isHighlight}" role="group">
```

Comment:
What is the intended purpose of `role="group"` here? Based on what I'm reading [on MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_group_role), I'm not sure this is appropriate for this case.